### DISCUSSION TITLE
Add resource-timegrid

### DISCUSSION THREAD

**Discussion by hahatulk** [saade/filament-fullcalendar#75] [Ideas]
Add please resource-timegrid to your project

https://fullcalendar.io/docs/vertical-resource-view

**Comment by saade** [comment]
https://github.com/saade/filament-fullcalendar/pull/66

**Reply by hahatulk** [reply]
> #66

thats is not the same, you added @fullcalendar/resource-timeline, I need @fullcalendar/resource-timegrid

**Reply by saade** [reply]
mb, are you able to PR it? I don't use this plugin, so i can't test it to see if it fulfills your requirements. it should be simple as #66, just two lines

**Reply by edeoliv** [reply]
> > #66
> 
> thats is not the same, you added @fullcalendar/resource-timeline, I need @fullcalendar/resource-timegrid

I also need this plugin /resource-timegrid

**Reply by hahatulk** [reply]
extend class and copy views, the only way for now =)

**Reply by edeoliv** [reply]
> extend class and copy views, the only way for now =)

Could you give me an example of how you achieved it?  because I tried it and wasn't able to get it to work.  Thanks in advance

**Reply by hahatulk** [reply]
Copy filament full calendar source to any folder to make it Independent from installed package. After that add needed plugins and configure them in js code of fullcalendar

**Reply by edeoliv** [reply]
Ok, thanks, I'll test it this afternoon

**Reply by saade** [reply]
Just make a PR, nothing is holding me from adding new plugins =)

**Reply by edeoliv** [reply]
@hahatulk unfortunately it did not work here

**Reply by hahatulk** [reply]
it is not as easy as sounds =)
Better make PR with added extension if you dont want to do everything manually

**Reply by jacf16** [reply]
I am new to using filament Laravel and I would like to know if it is not a bother how I could do it since I only get the day and the events, not the hours of the day vertically like the initial comment, I appreciate your support.
